core/pom.xml

```diff
@@ -468,6 +468,11 @@ THE SOFTWARE.
       <artifactId>commons-logging</artifactId>
     </dependency>
     <dependency>
+      <groupId>log4j</groupId>
+      <artifactId>log4j</artifactId>
+      <version>1.2.17</version>
```

Comment:
Most dependencies using properties, press plus in diff before this change. Also you can navigate upto https://github.com/jenkinsci/pom/blob/master/pom.xml#L611 but it ancient and variables mostly overrided in jenkins poms.